app/assets/src/stylesheets/00-base/_forms.scss

```diff
@@ -25,7 +25,7 @@ select,
 textarea {
   display: block;
   font-family: $base-font-family;
-  font-size: 16px;
+  font-size: 1rem;
```

Comment:
This appeared to be screwing font-things up but maybe not. Happy to change it back.